toplevel/coqloop.ml

```diff
@@ -266,7 +266,7 @@ let read_sentence ~state input =
   try Stm.parse_sentence ~doc:state.doc state.sid ~entry:G_toplevel.vernac_toplevel input
   with reraise ->
     let reraise = Exninfo.capture reraise in
-    discard_to_dot ();
+    if fst reraise <> Sys.Break then discard_to_dot ();
```

Comment:
could use a comment imo

Reply:
I added a comment. Please tell if ok for you.